unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -83,6 +91,11 @@ public override IEnumerator ApplyChanges(string newJson)
             avatarName.SetName(model.name);
             SetMinimapRepresentationActive(true);
             everythingIsLoaded = true;
+
+            avatarUserInfo.userId = model.id;
+            avatarUserInfo.userName = model.name;
+            avatarUserInfo.worldPosition = lastAvatarPosition != null ? lastAvatarPosition.Value : minimapRepresentation.transform.position;
+            UpdateAvatarIconInMinimap(avatarUserInfo);
```

Comment:
I would prefer a minimap-agnostic approach in the AvatarShape side (using AvatarShape events from some Minimap class), but i think that would be more difficult, so It's fine this way in my opinion.